.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+Welcome to the OpenTracing Specification repo
+
+- Please search for existing issues in order to ensure we don't have duplicate bugs/feature requests.)
+- Please be respectful and considerate of others when commenting on issues)
+- Please provide as much information as possible so we all understand the issue.)
+- Please don't ask questions here. If you have any question head to our gitter chat https://gitter.im/opentracing/public
+-->
+
+
+## Background
+Something that gives context about why this is an issue.
+
+## Problem
+Describe the problem. If the issue is about an improvement you can skip this. If possible, include a description of the impact of the problem.
+
+## Proposal
+A proposal that from your POV would solve the problem or improve the existing situation. It should also include the impact.
```

Comment:
I'd perhaps expand "POV" as we're in an international community, so abbreviations are sometimes hard